CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
 
 ## [Unreleased]
 
+### Changed
+
+- Add image registry switch
```

Comment:
Basically fine, but I feel like we should point out this should not be overwritten by the customer since we're setting this based on the installations region.

Reply:
I would put this in the `values.yaml` as a comment where quay.io is defined right now. Would you rather put this in the Changelog, or both?

Reply:
Agreed that it should be noted in the `values.yaml` as a comment.
I don't think it's necessary to mention it in the changelog.

Reply:
I will add it to the `values.yaml` and elaborate on the changelog entry to make it clear what the registry switch does